packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -337,15 +364,12 @@ class FloatingActionButton extends StatelessWidget {
     result = RawMaterialButton(
       onPressed: onPressed,
       elevation: elevation,
-      highlightElevation: highlightElevation,
+      highlightElevation: highlighElevation,
```

Comment:
Oops, should it be `highlightElevation: highlightElevation` (with `t` at the end of `highligh`)?

Reply:
thanks!